### ISSUE TITLE
Add the ability to select between multiple monitors in gaming mode

### ISSUE THREAD

**Issue by BioMyth** [ValveSoftware/SteamOS#1568]
#### Your system information

* Steam client version: 1719259903
* SteamOS version: 3.5.19
* Opted into Steam client beta?: [Yes/No] No
* Opted into SteamOS beta?: [Yes/No] No
* Have you checked for updates in Settings > System?: [Yes/No] Yes

#### Please describe your issue in as much detail as possible:
When in the following conditions occur: 

- In game mode
- Using the official steam deck dock
- With multiple monitors connected

There is no option to select which display I want to play my games on, the system defaults to the monitor connected via hdmi.

As I use the deck in game mode & desktop mode a lot this causes me to have to plug & unplug monitors frequently which can then cause issues with the steam deck & dock requiring reboots or unplugging everything from the dock and plugging it back in to fix.

#### Steps for reproducing this issue:

1. Setup the steam deck dock with two monitors 
2. Connect the steam deck to the dock in game mode
3. Note that the steam deck defaults to a specific monitor for output with no way of changing it

**Comment by BioMyth** [comment]
This is my first issue so let me know if it needs any changes!

**Comment by MrDynamo** [comment]
+1 for this issue. I have a 3 monitor setup connected to my dock. I can get it to use the far left only in desktop mode, but when in gaming mode it uses my far right (Far right and middle are reserved for work).

It would be great to be able to select which monitor you want to use in game mode rather than defaulting to one of the 3,

**Comment by Residays** [comment]
+1. I'd like to stay in gaming mode with the capture card connected and not have to constantly plug and unplug the hdmi cable.

**Comment by aesgeorges** [comment]
+1, also running a multi-monitor and see a lost opportunity on the steam deck screen to be useable with a dock.

**Comment by OrdinaryCloverOC** [comment]
i want multi monitor gaming. heard it's a feature in a batman game and wanna try it out.

**Comment by bmaximuml** [comment]
I've got 2 monitors and steamos gaming mode always picks the wrong one. I'd like to be able to tell it which screen to use without having to drop into desktop mode

**Comment by Nivisan-Jeyananthan** [comment]
I also have 2 external monitors 1080p, 1440p connected via dock it always picks the wrong one.  
I have tried the following method which **didn't work** for me. 
Maybe it does for someone else : 
https://www.reddit.com/r/SteamDeck/comments/17d9vem/swap_monitor_output_or_set_default_monitor_on/

In case the reddit post gets deleted from [Sabaku_no_Memo](https://www.reddit.com/user/Sabaku_no_Memo/) : 

> Solution: disconnect everything from your dock, connect power to it, then the deck, proceed to connect the monitor you don't want to be the main in gaming mode, wait for the image to appear, connect the monitor you want to be the main one in gaming mode. Reboot, restart at your heart's content.
TL:DR: basically the monitor you connect second is your main in gaming mode.

**Comment by regnets** [comment]
Got the same issue. I have got two monitors connected to my computer which uses steamos (so no traditional internal display).

One is my TV, the other one is my Monitor. I would like to switch between the TV and the Monitor depending on if I wanted to game on the couch or the desk. As others described only one monitor is used in gamescope. I would really appreciate to have the option to mirror or switch between those two.

**Comment by AbbyTheAbsol** [comment]
Now that SteamOS is on ARM, it'd be so cool to get multi-monitor support for these dual-screen handhelds that've been coming out. Please add this, Valve. I'd love to be able to check Discord on the bottom screen while playing a game on the top.

**Comment by Vysp3r** [comment]
Since we can now install SteamOS on any devices running an AMD GPU and the fact the Steam Machine will be available soon it would be a good idea to support multiple monitors.

**Comment by Seysix** [comment]
+1 I have this issue as well with my main monitor being connected through DP and it just defaults to my secondary monitor connected through HDMI

**Comment by OrdinaryCloverOC** [comment]
i feel more demand is coming with the steam machine.

**Comment by Kellegram** [comment]
Same issue here, my steam deck consistently chooses the only monitor I do not want to use, but I don't want to keep unplugging them because the ports will just wear out, I swapped docked devices many times a day.

**Comment by thomasstani** [comment]
Apologies for the duplicate - no idea how I missed this one on my search?

Anyway +1 on this! It’s hugely annoying that this isn’t a feature.

**Comment by anish-murthy-digipen** [comment]
+1, can we get this option?

**Comment by thesleepiestdev** [comment]
+1!

i duplicate between a 1440p@180hz DP monitor and a 1440p@60hz capture card, would love to be able to do this in gaming mode. i value 180hz as i am an avid lsfg-vk user!

EDIT: i want to add that for the past 2 weeks now, steamOS beta has this issue where whenever games try to skip the compositor, rendering freezes until i tab out or exit fullscreen. stable hasn't had this issue, but if this issue spreads to stable then i just won't be able to play at 180hz over displayport and duplicate to my capture card at 120 or 60hz, which would really suck because steam link and sunshine are both not stable enough to compete with a capture card :c

**Comment by Aelonath** [comment]
+1

Yes please. I use dual Monitors on my Steam Machine - DP for Main monitor, HDMI for side Monitor. I cant switch to my main Monitor in gaming mode ... unless i unplug the unwanted Monitor -.-
